taxonomies/food/ingredients.txt

```diff
@@ -49806,45 +49699,26 @@ wikipedia:en: https://en.wikipedia.org/wiki/Bogdanu%C5%A1a
 
 < en:varietal
 en: cabernet
+xx: cabernet
 de: Cabernet
 es: cabernet, vino cabernet
-fi: cabernet
-fr: cabernet
-hr: cabernet
-it: cabernet
-nl: cabernet
 # ingredient/fr:cabernet has 5 products @2019-01-19
 
 # description:en:Cabernet Franc is one of the major black grape varieties worldwide.
 
 < en:cabernet
-en: Cabernet Franc
-bg: Каберне Фран
-ca: Cabernet franc
-cv: Каберне фран
-de: Cabernet franc
-es: Cabernet franc
-et: Cabernet franc
-fi: Cabernet franc
-fr: Cabernet franc
+en: cabernet franc
+xx: cabernet franc
+bg: каберне фран
+cv: каберне фран
+de: Cabernet Franc
```

Comment:
idk if german wants both words here to be uppercased, i could see the argument of "franc" being an adjective to "Cabernet". my many years old german school knowledge is not good enough to know for this case. :)